osu.Game/Overlays/Chat/ChatLine.cs

```diff
@@ -236,7 +236,7 @@ public MenuItem[] ContextMenuItems
                 {
                     List<MenuItem> items = new List<MenuItem>
                     {
-                        new OsuMenuItem("View Profile", MenuItemType.Highlighted, Action)
+                        new OsuMenuItem("view profile", MenuItemType.Highlighted, Action)
```

Comment:
menu items are not tooltips, although I wouldn't mind making them lowercase (osu! text style is mostly in lowercase), but in a different PR please.